### PR TITLE
Adds Engine & Lookout Logo to Product Pages

### PR DESCRIPTION
**Pull request by ricardobaeta**
This solves https://github.com/src-d/landing/issues/365

Signed-off-by: Ricardo Baeta <ricardo@ricardobaeta.com>

### CODE REVIEW

**Comment by marnovo** [comment]
Please let's validate visually on desktop + mobile.

File: hugo/layouts/lookout/single.html [hunk 1]
```diff
@@ -16,7 +16,7 @@ <h1 class="title text-left p-0 mt-5 mb-4">Empower your Developers with source{d}
     <div class="container">
         <header class="text-center">
             <p class="subtitle text-orange">Product</p>
-            <h2 class="text-dark mb-5">source{d} Lookout</h2>
+            <img src="img/logos/sourced/sourced-lookout.svg" alt="source{d} Lookout Logo" width="316px" height="98px" />
```
Comment (dpordomingo) [inline hugo/layouts/lookout/single.html]:
```suggestion
            <img src="img/logos/sourced-lookout.svg" alt="source{d} Lookout Logo" width="316px" height="98px" />
```

File: hugo/layouts/engine/single.html [hunk 2]
```diff
@@ -16,7 +16,7 @@ <h1 class="title text-left p-0 mt-5 mb-4">Turn your code into actionable insight
     <div class="container">
         <header class="text-center">
             <p class="subtitle text-green">Product</p>
-            <h2 class="text-dark mb-5">source{d} Engine</h2>
+            <img src="img/logos/sourced/sourced-engine.svg" alt="source{d} Engine Logo" width="316px" height="98px" />
```
Comment (dpordomingo) [inline hugo/layouts/engine/single.html]:
```suggestion
            <img src="img/logos/sourced-engine.svg" alt="source{d} Engine Logo" width="316px" height="98px" />
```

**Comment by dpordomingo** [comment]
ready at staging

**Comment by ricardobaeta** [comment]
@marnovo With this change arises another thing to be tackled. We need to have consistent color coding on the product pages, which is not. And the logo itself needs some top & bottom margin, as seen on mobile. Lookout color coding currently being addressed at https://github.com/src-d/landing/pull/367#discussion_r262622124

image

image

**Comment by marnovo** [comment]
> **Please let's validate visually on desktop + mobile.**

@ricardobaeta 

Spacing on mobile…
Screen Shot 2019-03-15 at 1 58 02 PM

**Comment by ricardobaeta** [comment]
@marnovo 

From https://github.com/src-d/landing/pull/369#issuecomment-471522167 "And the logo itself needs some top & bottom margin, as seen on mobile."

**Comment by ricardobaeta** [comment]
@marnovo the latest commit addresses the correct mobile behavior.

**Comment by dpordomingo** [comment]
Already in staging.
Way better now, but still wondering if considering our rules about positive space, we should also leave more vertical space between `product` and `source{d} Lookout`
image

**Comment by ricardobaeta** [comment]
@marnovo @dpordomingo All good to go now!

**Comment by dpordomingo** [comment]
already at staging https://landing-staging.srcd.run/engine/
Can I deploy @ricardobaeta ?
image

**Comment by ricardobaeta** [comment]
Yes you can @dpordomingo  :)

**Pull request merged**